static/css/style.css

```diff
@@ -2,11 +2,19 @@
     fill: white;
 }
 
+#wrapper {
```

Comment:
There's already a `.container` class, just add `margin-top: 100px; margin-bottom: 100px` to it instead of making a new target.